src/test/connect_test.cpp

```diff
@@ -44,9 +44,8 @@
 
 double now()
 {
-  struct timeval tv;
-  gettimeofday( &tv, NULL );
-  return double(tv.tv_sec) + double(tv.tv_usec) / 1000000.0;
+  auto now = std::chrono::system_clock::now();
```

Comment:
Please use `std::chrono::steady_clock` as suggested [here](https://stackoverflow.com/a/18685338/4089003). Directly return the clock value (instead of double) and compute the duration where necessary.